tests/ui/macros/rfc-2011-nicer-assert-messages/all-expr-kinds.rs

```diff
@@ -107,77 +117,4 @@ fn main() {
     // unary
     [ -elem == -3 ] => "Assertion failed: -elem == -3\nWith captures:\n  elem = 1\n"
   );
-
-  // ***** Disallowed *****
-
```

Comment:
New AST variants were created and nobody bothered to include them here and probably nobody will in the future. It is then better to just test allowed expressions.